src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,57 +1,135 @@
 import React from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+
+// eslint-disable-next-line max-len
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.